product_docs/docs/migration_portal/3.5.0/04_mp_migrating_database/01_mp_schema_extraction.mdx

```diff
@@ -0,0 +1,210 @@
+   + `INCLUDE=` specifies which database object types to include in the extraction. Append a comma-separated list of the [Supported object types](#supported-object-types) to only extract database object types that are supported by Migration Portal. 
+
+
+4. From the command line, run the export command to generate a **db.dump** file. For example, to extract metadata information for `<Schema_1>`, `<Schema_2>`, `<Schema_3>`, and so on, run the following command:
+ 
+   ```
+   $ expdp <Username>@<ConnectIdentifier> DIRECTORY=DMPDIR SCHEMAS=<Schema_1>,<Schema_2>,<Schema_3> DUMPFILE=db.dump parfile=parameters.par
+   ```
+
+5. To generate a SQL file from the dump file, run the import command. 
+
+   For example, to generate **YourSchemas.SQL** file from the db.dump file, enter the following command:
+ 
+   ```
+    $ impdp <Username>@<ConnectIdentifier> DIRECTORY=<directory_name> TRANSFORM=OID:n SQLFILE=YourSchemas.sql DUMPFILE=db.dump
+   ```
+
+### To extract all schemas in a database
+
+!!! Note
+    Do not perform the following procedure from a user account that belongs to the excluded schemas list, see [Unsupported schemas](#unsupported-schemas). The impdp command will fail if the user account running the command is in the excluded list of schemas.
+
+
+1. In SQL*Plus, create a directory object that points to a directory on your server file system. For example:
+ 
+ **On Linux**:
+
+ `SQL> create directory DMPDIR as '/sample/file/path';`
+ 
+ **On Windows**:
+
+ `SQL> create directory DMPDIR as 'c:\sample\file\path\';`
+
+2. Grant read, write permissions to the user running the Data Pump utility.
+
+ `SQL> grant read,write on directory DMPDIR to <Username>;`
+
+3. Before running the expdp command, create a file with a `.par` extension (for example: parameters1.par) on your server. Add attributes and values to the file as shown below: 
+
+   ```
+   $ cat parameters1.par
+
+   CONTENT=metadata_only
+
+   FULL=Y
+
+   INCLUDE=SYNONYM,SEQUENCE,TYPE,TABLE,CONSTRAINT,PROCEDURE,FUNCTION,PACKAGE,INDEX,MATERIALIZED_VIEW,VIEW,TRIGGER,DB_LINK	
+ 
+   ```
+The attributes and values in the above command specify the following options:  
+   + `CONTENT=metadata_only` extracts only the metadata of the schemas.  
+   + `INCLUDE=` specifies which database object types to include in the extraction. Append a comma-separated list of the [Supported object types](#supported-object-types) to only extract database object types that are supported by Migration Portal.
+   + `FULL=Y` extracts all schemas in the database.
+
+
+4. From the command line, run the export command to generate a **db.dump** file, as follows:
+ 
+    ```
+    $ expdp <Username>@<ConnectIdentifier> DIRECTORY=DMPDIR DUMPFILE=db.dump parfile=parameters1.par
+    ```
+
+5. Before running the impdp command, create a parameter file with a `.par` extension (for example: parameters2.par) on your server. Add attributes and values to the file as shown below:
```

Comment:
```suggestion
5. Before running the impdp command, create a parameter file with a `.par` extension (for example, parameters2.par) on your server. Add attributes and values to the file:
```

Reply:
@prashantagar rightly pointed out that the command shows the contents of the file and does not add the contents. 

@drothery-edb i know we usually do not like using the word "below", so I'll replace it "as follows"